### DISCUSSION TITLE
Close your open issues/discussions please

### DISCUSSION THREAD

**Discussion by kRHYME7** [prasanthrangan/hyprdots#1478] [General]
Hi everyone! The discussion panel currently have too many unresolved discussions. 
Can you guys close your outdated discussions?  It's just too much to review one by one and would be unpolite to close it wihtout reviewing. 
 Thank you! 
Will do a cleanup next week.  😁

**Comment by rubiin** [comment]
I would help you out but unfortunately , it requires collaborator access
